etc/kayobe/kolla/globals.yml

```diff
@@ -5,8 +5,12 @@ docker_yum_gpgkey: "https://download.docker.com/linux/centos/gpg"
 
 {% if kolla_base_distro == 'centos' %}
 bifrost_tag: wallaby-20230207T194135
+blazar_tag: wallaby-20230303T172322
+caso_tag: wallaby-20230303T172322
```

Comment:
Caso image needs adding to stackhpc-release-train ansible/inventory/group_vars/all/kolla

Reply:
Thanks, adding here: https://github.com/stackhpc/stackhpc-release-train/pull/177